src/lex.js

```diff
@@ -114,10 +114,17 @@ window.Firth.lex = (function (Firth) {
                         break;
                     }
                 }
-                tokens.push({
-                    type: 'variable',
-                    name: token
-                });
+                if (token === 'true' || token === 'false') {
```

Comment:
I was thinking I'd prefer to make booleans just be variables (global constants `true`, `false`) if possible. I'm not sure though. It does make sense to have them be true literals, I guess.